### ISSUE TITLE
 Phone keeps asking to set outgoing calls to one sim 

### ISSUE THREAD

**Issue by NeoTheThird** [ubports/ubports-touch#40]
I have dual sim. Private and work and want to use both for outgoing calls.
When I place a call I get a pop up asking if I want outgoing calls always set to that sim.
I say no and check the box never ask again.
But on next call I get the pop up again

Reported by https://launchpad.net/~j-qander-r

**Comment by james-lyon** [comment]
Same for me with dual SIM on Aquaris E4.5

**Comment by NeoTheThird** [comment]
This might also be a general issue (eg. not device dependent)...

**Comment by james-lyon** [comment]
Yes, I seem to remember it being a very old bug that was sorted a long time ago. Apparently it has reappeared...

**Comment by NeoTheThird** [comment]
@Flohack74 Interesting, so it does depend on the device... @james-lyon Do you have a link to the commit that fixed it or the original bug-report?

**Comment by Flohack74** [comment]
Is this still an issue?

**Comment by james-lyon** [comment]
The issue still exists for me on Aquaris E4.5 


On Mon, 25 Dec 2017, at 10:43 AM, Florian Leeber wrote:
> Is this still an issue?


> — You are receiving this because you were mentioned. Reply to this
> email directly, view it on GitHub[1], or mute the thread[2].> 


Links:

  1. https://github.com/ubports/ubuntu-touch/issues/40#issuecomment-353855034

**Comment by james-lyon** [comment]
A year later, on a fresh install of OTA-7, I am still facing this problem. 


The only 'workaround' I have found to avoid getting the dialog box each time I make a call is to set one SIM to use by default.

**Comment by theron29** [comment]
I can confirm this issue on two devices of mine:
- E5HD /16.04 2019-W24/
- Meizu PRO5 /16.04 2019-06-12/
I had this issue on E5HD for a very long time...

**Comment by kwah-gh** [comment]
Seems to work fine here. FP2. Dual SIM. Maybe not everyone knows about preferences part on call selection?

**Comment by james-lyon** [comment]
@kwah-gh when you say "works fine" do you have one SIM set as default? The bug only appears if you have **no SIM** set as the default and check the box never to ask again.

**Comment by Flohack74** [comment]
Ok so I now understood the problem really: I always had a default SIM set for all options so I never run into that issue. So yeah, I can can see thats probably a persistency issue with the option "Never ask again"...
